src/psyclone/tests/domain/lfric/transformations/lfric_extract_test.py

```diff
@@ -128,19 +128,29 @@ def test_distmem_error():
             "not supported.") in str(excinfo.value)
 
     # Try applying Extract transformation to Node(s) containing HaloExchange
+    # We have to disable distributed memory, otherwise an earlier test
+    # will be triggered
+    from psyclone.configuration import Config
+    config = Config.get()
+    config.distributed_memory = False
     with pytest.raises(TransformationError) as excinfo:
         _, _ = etrans.apply(schedule.children[2:4])
     assert ("Nodes of type '<class 'psyclone.dynamo0p3.DynHaloExchange'>' "
             "cannot be enclosed by a LFRicExtractTrans "
             "transformation") in str(excinfo.value)
+    config.distributed_memory = True
 
     # Try applying Extract transformation to Node(s) containing GlobalSum
     _, invoke = get_invoke("15.14.3_sum_setval_field_builtin.f90",
                            DYNAMO_API, idx=0, dist_mem=True)
     schedule = invoke.schedule
     glob_sum = schedule.children[2]
+    # We have to disable distributed memory, otherwise an earlier test
+    # will be triggered
+    config.distributed_memory = False
     with pytest.raises(TransformationError) as excinfo:
         _, _ = etrans.apply(glob_sum)
+    config.distributed_memory = True
```

Comment:
Can we use monkeypatch to modify `config` instead?

Reply:
Done.